### PR TITLE
useraccount ctrl added with noop

### PR DESCRIPTION
**Pull request by nurali-techie**
Fix: https://jira.coreos.com/browse/CRT-133

### CODE REVIEW

File: pkg/controller/useraccount/useraccount_controller.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+	// Watch for changes to primary resource UserAccount
+	err = c.Watch(&source.Kind{Type: &toolchainv1alpha1.UserAccount{}}, &handler.EnqueueRequestForObject{})
+	if err != nil {
+		return err
+	}
+
+	// TODO(user): Modify this to be the types you create that are owned by the primary resource
+	// Watch for changes to secondary resource Pods and requeue the owner UserAccount
+	// err = c.Watch(&source.Kind{Type: &corev1.Pod{}}, &handler.EnqueueRequestForOwner{
```
Comment (alexeykazakov) [inline pkg/controller/useraccount/useraccount_controller.go]: The secondary resources for this controller are Users and Namespaces and probably ProjectRequets...

Comment (nurali-techie) [reply pkg/controller/useraccount/useraccount_controller.go]: Yup, that will be add when we start creating kube_user etc in reconcile.

Comment (MatousJobanek) [reply pkg/controller/useraccount/useraccount_controller.go]: Could you please remove the lines that are commented out? 

> The secondary resources for this controller are Users and Namespaces and... 

Not Namespaces, but NsTemplateSet.

**Comment by nurali-techie** [comment]
@MatousJobanek this is very basic impl and it will go through lot of changes.

**Pull request merged**